pkg/cluster/sync.go

```diff
@@ -2,6 +2,7 @@ package cluster
 
 import (
 	"fmt"
+
```

Comment:
Just for the records - I think it makes sense to avoid an unnecessary changes next time.